packages/lab/src/layout/LayerLayout/LayerLayout.css

```diff
@@ -0,0 +1,110 @@
+.uitkLayerLayout {
```

Comment:
Maybe we should have a `@media (prefers-reduced-motion)` media query  to disable the slide animations?

Reply:
This was suggested for the ParentChild component as well but then we've agreed to add that as part of the storybook example. I can add an example for this component as well if we think that's the best solution? Or should we add @media (prefers-reduced-motion) to both components?

Reply:
Interesting, so we're leaving it up to the consumer to add reduced motion.

I think for me it would be nice to have that built into the components at least for slide animations, but maybe a good one to discuss with the group on our next standup